esm-samples/jsapi-angular-cli/README.md

```diff
@@ -5,73 +5,45 @@ This repo demonstrates how to use [@arcgis/core](https://www.npmjs.com/package/@
 ---
 **Known issues**
 
-* It's currently recommended to set `buildOptimizer: false` in `angular.json`. This prevents WebGL errors such as `WebGL: INVALID_OPERATION` and `WebGL: INVALID_ENUM`. It may increase your on-disk and loaded application size.
+* We are aware of a general issue with `Unhandled Promise Rejection` errors. They aren't specific to Angular and shouldn't affect ArcGIS JS API functionality. Upgrade to `11.2.5`+ and switch the `tsconfig.target` to es2017.
```

Comment:
- remove "aren't specific to Angular and " or make a distinction between unhandled abort errors and Angular only Zone.js errors
- "Upgrade to `11.2.5`+ and switch the `tsconfig.target` to es2017"
  - Maybe "Upgrade Angular to `11.2.5`+ and switch the `tsconfig.target` to `es2017`+"
  - Make clearer that this will prevent the `Unhandled Promise Rejection` errors that occur only in Angular when using Zone.js

Reply:
Done.